src/pid.cpp

```diff
@@ -179,16 +179,20 @@ double Pid::compute_command(
 
 double Pid::compute_command(double error, double error_dot, const double & dt_s)
 {
+  // don't update anything
+  if (dt_s <= 0.0) {
+    return cmd_;
+  }
+
```

Comment:
I would raise an exception if the dt_s is negative 

And when it is zero, cmd

Reply:
Because negative time should never happen in a monotonic slewless clock, and atleast with an exception the user will know that there is something wrong with his clock

Reply:
done